src/plugins/intel_cpu/tests/functional/shared_tests_instances/snippets/exp.cpp

```diff
@@ -0,0 +1,38 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "snippets/exp.hpp"
+#include "common_test_utils/test_constants.hpp"
+
+namespace ov {
+namespace test {
+namespace snippets {
+
+
+namespace {
+// ===================================Exp=========================================================//
+std::vector<ov::test::InputShape> inShapesStatic{{{}, {{1, 1, 32, 128}}}};
+
+INSTANTIATE_TEST_SUITE_P(smoke_Snippets_Eltwise, Exp,
```

Comment:
Should be `smoke_Snippets_Exp` maybe?

Reply:
A lot of simple eltwise tests are handled inside the `smoke_Snippets_Eltwise` suite, like [here](https://github.com/IvanNovoselov/openvino/blob/1e8fbd3bb2bcac4500811b0afe31391172ea6c5a/src/plugins/intel_cpu/tests/functional/shared_tests_instances/snippets/add.cpp#L84-L119)

Let's create a separate suite for the Exp, I like a more fine-grained division too